### ISSUE TITLE
Upload Code

### ISSUE THREAD

**Issue by ntquyen11** [hadi-ghnd/AccidentDetection#1]
Can you upload your code?

**Comment by hadign20** [comment]
Yes, right now there is a limitation as the project is ongoing, but we will upload the code shortly (maximum 1 month from now).

**Comment by O-Hatem** [comment]
Is the code available right now?

**Comment by DonghanChen** [comment]
Hi, could you share the code?

**Comment by Volcano152** [comment]
Hello, would you mind sharing your code?

**Comment by saigowtham627** [comment]
Hi, would you mind sharing the code ?
